app.js

```diff
@@ -0,0 +1,14 @@
+const menuHamburger = document.querySelector('.icon-menu');
+const navLinks = document.querySelector('.links');
+
+menuHamburger.addEventListener('click', () => {
+  navLinks.classList.toggle('mobile-menu');
+});
```

Comment:
- [ ] When the user clicks on the `Porfolio` link, the menu should dissapear and the user should be taken to the projects part of the page. Currently the menu is not dissapearing. Kindly fix this.
![image](https://user-images.githubusercontent.com/81932478/224852232-1aed2924-c62f-4d82-b3c3-8a159c14ed8a.png)